style.css/styles.css

```diff
@@ -253,16 +253,18 @@ padding-right: 5%;
 
 .bg2 {
   position: absolute;
+  height: 500px;
   top: 0%;
   left: 0px;
   z-index: -1;
 }
 
 .bg3 {
   position: absolute;
-  bottom: 0%;
+  height: 510px;
```

Comment:
In the UI design, this plant is suppose to be by the laptop image in the .privacy class not at by the grid at the top

Reply:
![Screenshot (91)](https://user-images.githubusercontent.com/66262180/150965999-bb722fdb-2eed-495e-84b1-0ef895f78b33.png)


The design there now has the plant at the top

Reply:
Ok